src/runtime/debug/arc-debug-handler.ts

```diff
@@ -16,6 +16,8 @@ import {Particle} from '../recipe/particle.js';
 import {ArcStoresFetcher} from './arc-stores-fetcher.js';
 import {DevtoolsConnection} from './devtools-connection.js';
 import {enableTracingAdapter} from './tracing-adapter.js';
+import { Slot } from '../recipe/slot.js';
+import { stringify } from 'querystring';
```

Comment:
Nit: please remove the inner spaces; seems most auto-format tools want to add them but the vast majority of our code base doesn't have them :-/